### PR TITLE
Added IT language

### LINKED ISSUE #34: Wrong locales encapsulation in skill.json
en-IN and pt-BR in `skill.json` are out from `locales` object

### PR DESCRIPTION
**Pull request by acerbisgianluca**
Changes:
- Added it_IT language in models, resources and skill.json.
- Ran `npm run lint`.
- Fixed #34.

### CODE REVIEW

File: skill.json [hunk 1]
```diff
@@ -1,31 +1,41 @@
...
-  }
+    "manifest": {
+        "publishingInformation": {
+            "locales": {
+                "en-US": {
+                    "summary": "Alexa Skill For Rocket.Chat",
+                    "examplePhrases": [
+                        "Alexa, Open Rocketchat",
+                        "Alexa, Begin Rocketchat",
+                        "Alexa, Begin Rocketchat"
```
Comment (PrajvalRaval) [inline skill.json]: Please change "Alexa, Begin Rocketchat" to "Alexa, Launch Rocketchat"

**Review by PrajvalRaval — CHANGES_REQUESTED** [review]
Your Intent Schema looks fine but you cannot keep the slot values for "message" in English. You will need to remove english slot values and add italian slot values instead in order to make that slot work in italian language.

File: skill.json (hunk 1 above)
Comment (acerbisgianluca) [reply skill.json]: Of couse, thank you

**Comment by acerbisgianluca** [comment]
When merged this should close #12

**Review by TheDreamSaver — APPROVED** [review]

**Review by PrajvalRaval — APPROVED** [review]

**Pull request merged**